### PR TITLE
Issue 13653 edit mode ui fixes round 2

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts [hunk 1]
```diff
@@ -253,4 +251,10 @@ export class DotEditLayoutGridComponent implements OnInit, ControlValueAccessor
     private isHaveRows(): boolean {
         return !!(this.value && this.value.rows && this.value.rows.length);
     }
+
+    private resizeGrid(timeOut?): void {
+        setTimeout(() => {
+            this.ngGrid.triggerResize();
+        }, timeOut ? timeOut : 0);
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts]: Are you sure we want ```0``` here? https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful

File: src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts [hunk 2]
```diff
@@ -100,8 +100,9 @@ export const EDIT_PAGE_CSS = `
 
     .dotedit-container__toolbar {
         position: relative;
-        margin: -20px 8px 0 0;
+        margin: -15px 8px 0 0;
         float: right;
+        font-size: 0;
```
Comment (fmontes) [inline src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts]: What is this for?

File: src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts (hunk 1 above)
Comment (hmoreras) [reply src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts]: yes. there are times when we don't want to wait more than the time effect that creates the timeout

File: src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts (hunk 2 above)
Comment (hmoreras) [reply src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts]: display the `+` button correctly in the containers, for empty templates or any other.

**Pull request merged**